plugins/node/opentelemetry-instrumentation-mongodb/src/types.ts

```diff
@@ -43,6 +56,13 @@ export type CursorState = { cmd: MongoInternalCommand } & Record<
   unknown
 >;
 
+// https://github.com/mongodb/node-mongodb-native/blob/3.6/lib/core/connection/command_result.js
+export type CommandResult = {
+  result?: string;
+  connection?: string;
+  message?: string;
```

Comment:
Why are these typed as `string`s? From the link you are referencing it looks like at least `result` and `connection` are objects

Reply:
they are received as strings in this context

Reply:
are they received as `string` from the driver itself?
The tests try to `JSON.stringify` this value when used, so something is not consistent

Reply:
Changed to `object`! :)